doc/Type/Metamodel/CurriedRoleHOW.pod6

```diff
@@ -61,6 +61,10 @@ a single parameter an instantiated role will also be of the same type:
     role Zape[::T] {};
     say Zape[Int].HOW; #: «Perl6::Metamodel::CurriedRoleHOW.new␤»
 
+Curried roles L<pretend to be|/type/Metamodel::TypePretense> of types
```

Comment:
I really appreciate your work here, and in general. The problem is that it takes a long time to review a PR with 8 files. So, this is not really something you should react to, but in this specific case it would have been much better if files like this would have gone to a separate PR, or just submitted directly. After all, they're essentially a different issue. In that case, I have to check that the files you're linking are the ones that have been already included in this issue, so it's additional work...